Algorithms/Balanced_Parenthesis.cpp

```diff
@@ -0,0 +1,62 @@
+#include <iostream>
+#include <stack>
+#include <string>
+using namespace std;
+bool check(char x, char y)
+{
+	if ((x=='(' && y==')') || (x=='{' && y=='}') || (x=='[' && y==']') || (x=='<' && y=='>'))
+	{
+		return true;
+	}
+	else
+	{
+		return false;
+	}
+}
+
+bool checkbalancedparenthesis(string s)
```

Comment:
Please name the functions according to the [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards) naming conventions. 

Also, improve the entire code following the conventions given in the above wiki page.